### PR TITLE
Instructor view results: increase responses limit to 8000 #6586

### LINKED ISSUE #6586: Instructor view results: increase responses limit to 8000
Currently, a result page is not loaded if it involves more than 2000 responses. This limit is too low. Let's increase it to 8000.
The limit is defined in FeedbackSessionsLogic.java

### PR DESCRIPTION
**Pull request by aigyz**
Fixes #6586

### CODE REVIEW

File: src/main/java/teammates/logic/core/FeedbackSessionsLogic.java [hunk 1]
```diff
@@ -839,7 +839,7 @@ public String getFeedbackSessionResultsSummaryInSectionAsCsv(
             throws EntityDoesNotExistException, ExceedingRangeException {
 
         FeedbackSessionResultsBundle results;
-        long indicatedRange = section == null ? 2000 : -1;
+        long indicatedRange = section == null ? 8000 : -1;
```
Comment (Neurrone) [inline src/main/java/teammates/logic/core/FeedbackSessionsLogic.java]: Make this a constant in const.java

**Comment by aigyz** [comment]
Hi, @Neurrone.
Made it as a constant.

**Comment by Neurrone** [comment]
@aigyz thanks for your contribution, and good work following our workflow process without any mistakes :)

**Comment by aigyz** [comment]
@Neurrone great, thanks for your reviews.
Have a good day :)

**Pull request merged**